lib/pbench/server/api/resources/query_apis/datasets/table_of_content.py

```diff
@@ -39,9 +39,12 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
 
         Args:
             json_data: JSON dictionary of type-normalized parameters
+            context: propagate the requested set of metadata to the
+                    postprocess method.
                 "run_id": Dataset document ID
 
         EXAMPLE:
+        We read "run_id" from 'context'
         {
             "run_id": "1234567890"
             "parent": '/default'
```

Comment:
I suggest approaching this from a more this-function-centric perspective.

There are two arguments, `json_data` and `context` -- what does _this function_ use them for?  (Answer:  we pull the value of the `"parent"` key from `json_data` and the value of the `"run_id"` key from `context`...and that's **it**, right?)  It's like the actual parameters to this function are the `parent` and `run_id` values...so I would be inclined to document that, rather than getting hung up on the greater purpose and uses of the `json_data` and `context` parameters.  (Does that make sense?)